cfme/tests/cli/test_appliance_console.py

```diff
@@ -0,0 +1,159 @@
+import pytest
+from wait_for import wait_for
+from utils import version
+from utils.log_validator import LogValidator
+
+
+def test_black_console_set_hostname(request, appliance):
+    hostname = 'test.com'
```

Comment:
I would propose using something like "something.redhat.com" to avoid using an owned fqdn